VBproject/VBproject/views.py

```diff
@@ -13,40 +20,88 @@ def create_view(request: HttpRequest) -> HttpResponse:
     return HttpResponse('FORM.')
 
 
-def article_view(request: HttpRequest, article: str) -> HttpResponse:
-    return HttpResponse(f'Default article - {article}.')
+def article_view(request: HttpRequest, title_article: str) -> HttpResponse:
+    try:
+        article_obj: Article = Article.objects.get(title=title_article)
+    except Article.DoesNotExist:
+        raise Http404('Article with this title does not exist.')
 
+    comments: list = Comment.objects.filter(article_comment__title=article_obj)  # querySet
```

Comment:
```suggestion
    comments: QuerySet = article_obj.comments.all() 
```
Треба подивитись як прописані бекрефи у моделях, але має бути щось схоже.